### ISSUE TITLE
what does the pretrain=True flag do in the mini-imagenet data loader?

### ISSUE THREAD

**Issue by brando90** [WangYueFt/rfs#16]
@WangYueFt  @HobbitLong  what does the `pretrain=True` flag do? 

e.g.

```
        if self.pretrain:
            self.file_pattern = 'miniImageNet_category_split_train_phase_%s.pickle'
        else:
            self.file_pattern = 'miniImageNet_category_split_%s.pickle'
```

https://github.com/WangYueFt/rfs/blob/f8c837ba93c62dd0ac68a2f4019c619aa86b8421/dataset/mini_imagenet.py#L42

**Comment by txw1997** [comment]
Hello! Have you trained on the tieredImageNet yet? Can you share some information about the validation accuracy when training on the trainset? 
My validation accuracy just 0.5%, but training accuracy is gradually ascending, do you have the same issue?
